spec/helpers/peep.rb

```diff
@@ -0,0 +1,9 @@
+module PeepHelpers 
+
+ def post(peep)
+    visit'/peeps/new'
```

Comment:
Put space between the method name and the first argument.<br>Use 2 (not 3) spaces for indentation.